manager/allocator/network.go

```diff
@@ -231,7 +231,9 @@ func (a *Allocator) doNetworkInit(ctx context.Context) (err error) {
 
 	var allocatedServices []*api.Service
 	for _, s := range services {
-		if nc.nwkAllocator.IsServiceAllocated(s) {
+		if nc.nwkAllocator.IsServiceAllocated(s, func(options *networkallocator.ServiceAllocationOpts) {
+			options.OnInit = true
+		}) {
```

Comment:
Please define this closure as a function in networkallocator.go. Then you can write something like:

`if nc.nwkAllocator.IsServiceAllocated(s, OnInit) {`